packages/qwik/src/server/prefetch-implementation.ts

```diff
@@ -35,35 +33,30 @@ export function includePreloader(
     }
   }
 
-  const makeLink = (base: string, href: string) => {
-    const linkProps: PropsOf<'link'> = {
-      rel: linkRel!,
+  /** We only use modulepreload links because they behave best. Older browsers have the preloader. */
```

Comment:
what does this mean? The preloader will be enough for older browsers? how?

Reply:
No, the small % of older browsers will ignore SSR preloads, and for them the preloader does `preload` instead of `modulepreload` once it runs.

Since the default is now 0 ssr preloads, there is no effect.